app/src/context/DocumentStore.tsx

```diff
@@ -52,6 +54,10 @@ export const DocumentProvider: React.FC<{
       Crypto.CryptoDigestAlgorithm.SHA256,
       file.base64
     );
+    
+    let buffer = Buffer.from(file.base64, "base64");
```

Comment:
Could this be a const?

```suggestion
    const buffer = Buffer.from(file.base64, "base64");
```